testgrid/cmd/summarizer/README.md

```diff
@@ -0,0 +1,48 @@
+# Testgrid Summarizer
+The Testgrid Summarizer creates test result summaries for the Testgrid
+dashboards and tables. It continuously writes summaries owned by the
+Update Master.
+
+## Workflow
+The `Update Master` will parse all the groups and request the `Update Servers` to aggregate test results. When the update cycle is done, the `Update Master` will run post-update jobs, which will trigger the `Summerizer` to create the summary object.
+
+## Highlevel Design
+The `Summarizer` implements the key functions to convert aggregated results into the appropriate summary format, and continuously writes/reads summaries owned by the `Update Master` to/from storage. This will be a standalone service for all flavor of Testgrid clusters to use for test result translation. The summarizer server will implement the gRPC server component.
+
+```
+type Summarizer struct {}
+func (s *Summarizer) TableToSummary(...) (...) {}
+func (s *Summarizer) SummaryToClientObject(...) (...) {}
+func (s *Summarizer) GetStatusForDashboard(...) (...) {}
+
+type Storage struct{}
+func (s *Storage) SaveSummaryObject() {}
+func (s *Storage) RestoreSummaryObject() {}
+func (s *Storage) NormalizeObjectName() {}
+
+type SummarizerServer struct {}
+func (s *SummarizerServer) GetSummary(...) (...) {}
+func (s *SummarizerServer) GetDashboardStatus(...) (...) {}
+```
+
+## Implementation Breakdown
+The Summarizer component will be implemented in two stages:
```

Comment:
Which parts of this implementation does this PR resolve?

Reply:
This PR implements stage 1. When the reference client makes a RPC call, it will receive the DashboardTabSummary, which is the translated summary. This PR does not store the result.

Reply:
Can you note that in the readme? The current statement is misleading. Perhaps note that this component does X, but not Y

Alternately, strike the section entirely. I want to avoid an out-of-date readme

Reply:
Thanks, added the PR number to the breakdown to make it clear. This is to help the reviewers, as suggested by Katharine and Aaron. I will remove this section completely once the summarizer is done.